android/README.md

```diff
@@ -0,0 +1,96 @@
+# Liferay Screens for Android
+
+## Important Note
+
+*This product is under heavy development and its features aren't ready for use in production. It's being made public only to allow developers to preview the technology*.
+
+## Introduction
+
+Liferay Screens for Android includes the component (*screenlets*) library and a sample project. In Liferay Screens, a screenlet is a visual component that is connected to Liferay Portal's functionality. Screenlets use a specific UI to handle input from the user and present information from the portal. This enables the app to receive user input, handle communication with the server, and present the user with the results.
+
+Each screenlet is tied to one or more of [Liferay's remote services](https://www.liferay.com/documentation/liferay-portal/6.2/development/-/ai/accessing-services-remotely-liferay-portal-6-2-dev-guide-05-en). When the screenlet needs to show information to the user, it relies on a *view*. View sets can be contributed by third parties and are fully pluggable, so you aren't limited to one specific look and feel. The Screens library provides the standard *Default* view set and the sample *Material* view set.
+
+<!-- TODO Cool picture with android phone -->
+
+![The SignUp screenlet using the Default and Flat7 themes.](Documentation/Images/screens-phone2.png)
+
+To learn about the architecture of Screens for Android, see the [library documentation page.](library/README.md).
+
+## Requirements
+
+Development of Android apps using Liferay Screens requires the following: 
+
+- Android Studio 1.0.2 or above
+- Android SDK 4.0 (API Level 14) or above
+<!-- - Gradle??? -->
+- [Liferay Portal 6.2 CE or EE](http://www.liferay.com/downloads/liferay-portal/available-releases)
+- [Liferay Screens' compatiblity plugin](https://github.com/liferay/liferay-screens/tree/master/portal). 
+- Liferay Screens' source code
+
+## Compatibility
+
+Liferay Screens for Android is compatible with Android 4.0 (API Level 14) and higher. 
+<!-- TODO: Mention compat library??? -->
+
+## Preparing Your Project for Liferay Screens
+
+Liferay Screens is released as an [AAR file](http://tools.android.com/tech-docs/new-build-system/aar-format) in the Maven Central repository. You therefore need to use Maven or Gradle to set and download your dependencies. Use the following steps to configure your project with Gradle:
+
+TODO
+1. XXX
+2. XXX
+3. XXX
+
+Great! Your project should now be ready for Liferay Screens. Next, you'll learn how to use screenlets in your app.
+
+## Using Screenlets
+
+Now that your project is configured to use Screens, you can start using screenlets. First, insert a screenlet in your activity or fragment layout. You can do this in XML or with the visual layout editor in Android Studio.
+
+![Insert a screenlet.](documentation/images/insert_screenlet.png)
```

Comment:
The root layout of this image is a FrameLayout, so it should not have an attribute "orientation".
For sake of simplicity, the "tools" namespace and "tools" attribute could be removed from the FrameLayout.